src/config_logs/cloud_sql.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cloud_sql",
+  "displayName": "Cloud SQL",
```

Comment:
this name doesn't mean which kind of DB it is, mysql, postgresql or sql server

Reply:
yes, beacause we don't have anythink in the GCP logs for cloud_sql that we can use to detect engine; this config will be common for all eninges of cloud sql